test/js/bun/ffi/ffi.test.js

```diff
@@ -641,3 +642,294 @@ it(".ptr is not leaked", () => {
     expect(fn.ptr).toBeUndefined();
   }
 });
+
+const test = existsSync("/lib/x86_64-linux-gnu/libc.so.6") ? it : it.skip;
+
+test("can open more than 63 symbols", () => {
+  const lib = dlopen("/lib/x86_64-linux-gnu/libc.so.6", {
```

Comment:
isnt there a version of this on MacOS?

Reply:
just added, also remove/re-add other symbols to be compatible